.build/ci/oracle-setup.sh

```diff
@@ -4,34 +4,34 @@ export SQLPATH=./.build/ci
 printf "\nSetup database...\n"
 
 # we hebben meer dan de default 300 processen nodig
-docker exec -i oracle_brmo sqlplus -l sys/oracle@//localhost:1521/XE as sysdba < ./.build/ci/oracle-system-setup.sql
+docker exec -i oracle_brmo sqlplus -l sys/oracle@//localhost:1521/FREE as sysdba < ./.build/ci/oracle-system-setup.sql
```

Comment:
We zullen vooralsnog compatibility level op 21.0.0 moeten instellen om BOOLEAN datatype uit te schakelen middels `ALTER SYSTEM SET COMPATIBLE='21.0.0' SCOPE = SPFILE;`

Reply:
zie ook https://github.com/gvenzl/oci-oracle-free/issues/41